package.json

```diff
@@ -38,7 +38,7 @@
     "build": "lerna run --loglevel=silent build",
     "build:current": "lerna run --loglevel=silent build:current",
     "pretest": "npm run build:current && npm run cli:link",
-    "cli:link": "cd packages/cli/generators && npm link && cd ../../..",
+    "cli:link": "cd packages/cli/generators/app && npm link && cd ../controller && npm link && ../extension && npm link && ../project && npm link && cd ../../..",
```

Comment:
This is completely wrong. Can you explain?

Reply:
Please note we don't depend on global installation of generators as `@loopback/cli` does not use `yo`.

Reply:
I see. I'll have to see if I can get my hands on a windows machine as I see a lot of tests failing on AppVeyor with the following message: 

```
Error: You don't seem to have a generator with the name “loopback4:C:\projects\loopback-next\packages\cli\generators\app” installed.
But help is on the way:
You can see available generators via npm search yeoman-generator or via http://yeoman.io/generators/. 
Install them with npm install generator-loopback4:C:\projects\loopback-next\packages\cli\generators\app.
To see all your installed generators run yo without any arguments. Adding the --help option will also show subgenerators. 
If yo cannot find the generator, run yo doctor to troubleshoot your system.
      at Environment.create (packages\cli\node_modules\yeoman-environment\lib\environment.js:368:9)
      at Object.exports.testSetUpGen (packages\cli\test\test-utils.js:15:14)
      at Context.it (packages\cli\test\base-generator.js:15:29)
```